examples/dashboard.json

```diff
@@ -18,6 +18,65 @@
   "id": 12,
   "links": [],
   "panels": [
+    {
+      "type": "stat",
+      "title": "up(controller)",
+      "gridPos": {
+        "x": 0,
+        "y": 0,
+        "w": 3,
+        "h": 4
+      },
+      "id": 32,
+      "targets": [
+        {
+          "expr": "up{kubernetes_pod_name=~\"argo-rollouts-.*\"}",
```

Comment:
I believe  `kubernetes_pod_name` presumes user is running `kube-state-metrics` which probably isn't a safe assumption to make in all clusters.

In Argo CD, we rely on an "uptime" gauge, which is based on `process_start_time_seconds `, which is a standard prometheus metric I think already exported by rollout controller:
https://prometheus.io/docs/instrumenting/writing_clientlibs/#process-metrics

![image](https://user-images.githubusercontent.com/12677113/123319655-8b9f3c00-d4e5-11eb-834b-d3594d30dfa0.png)

WDY think about a singlestat panel that shows uptime, instead of this?